proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+
+SCHOOLS
+
+Informatics, (New) Media
+
+Curriculum has to be accompanied by teaching skills in and understanding of digital and especially cryptographic tools (open source alternatives to basic software like OSs, alternatives to commercial service providers like GAFAM, browser extension or crypto-browsers, tool to encrypt communication like encrypted mailing, messaging, password managers etc.)
+
+<lynX> If we let humanity live on in the mine field of an unregulated Internet, then we need to spend billions in educating the teachers so that they actually know what they are talking about. Probably easier to change the Internet so that it no longer is a mine field. Then *nobody* has to learn encryption tools, and *nobody* has to understand why they are necessary. The "alternative" would become the default as mandated by legislation (#ObCrypto).
+
+Social science
+
+Curriculum has to offer a focus media science that is up to date to accompany the practical skills. Lessons in Philosophy of technology where schools don't have philosophy classes.
+
+Art
+
+A focus on New Media has to be offered. E. G. teaching kids to cut videos to gain an understanding of how media are essentially montages. Elaborating on skills and methods in a creative way.
+
+PUBLIC SECTOR
+
+Institutions
+
+Public places and institutions are likewise able to foster and implement more democratic technologies. For instance libraries can set up their systems in a more open source secure way while at the same time spreading knowledge and best practices to its users. A good example for this is the American Library Freedom Project (https://libraryfreedomproject.org/)
+
+Public media, public advertisement
+
+Another place to push back the dominance of corporate power and abuse of digital technologies by other players is to oppose their all encompassing presence in media and advertising (TV, Internet, Public sphere). There have to be state- or EU funded campaigns advocating for the above mentioned solutions and their essential importance for a free, democratic and also economically more just society - think platform and surveillance capitalism where, corporations mine data to generate exorbitant surplusses, without paying taxes and drastically changing “the game” for unaware citizens respectively, in corporate talk, users.
```

Comment:
We don't want to censor the (privatized) public sphere, right? What could be a more moderate means to ensure an ad-free public discourse? - subliminal guy

Reply:
Let's forbid unethical surveillance capitalism rather than de-facto accepting it by running (desperate) campaigns for the "alternatives." - lynx